src/Avalonia.Controls.DataGrid/DataGridCell.cs

```diff
@@ -173,7 +173,7 @@ private void DataGridCell_PointerPressed(PointerPressedEventArgs e)
                     }
                     if (OwningRow != null)
                     {
-                        e.Handled = OwningGrid.UpdateStateOnMouseLeftButtonDown(e, ColumnIndex, OwningRow.Slot, !e.Handled);
+                        OwningGrid.UpdateStateOnMouseLeftButtonDown(e, ColumnIndex, OwningRow.Slot, !e.Handled);
```

Comment:
Hard to say if it's breaking change or not.
If touch input is handled by cell selection, then it's not possible to scroll content.

Reply:
tis kinda weird to scroll inside cells anyway so probably not

Reply:
Well, in this case it's about "pointer pressing". Scroll gestures listen for this event to initiate a gesture.
But if this event was already handled somewhere, it doesn't scroll cell's parents (including data grid).

Reply:
We might want to check the pointer type and keep the previous logic when mouse is used.